trieste/acquisition/function.py

```diff
@@ -137,9 +137,9 @@ def expected_improvement(model: ProbabilisticModel, eta: tf.Tensor, at: QueryPoi
     return (eta - mean) * normal.cdf(eta) + variance * normal.prob(eta)
 
 
-class MaxValueEntropySearch(SingleModelAcquisitionBuilder):
+class MinValueEntropySearch(SingleModelAcquisitionBuilder):
     """
-    Builder for the max-value entropy search acquisition function (for function minimisation)
+    Builder for the min-value entropy search acquisition function (for function minimisation)
```

Comment:
can you add something explaining this is the Max value entropy search algorithm modified for function minimisation?